src/workSchedules/types.ts

```diff
@@ -10,7 +10,9 @@ export type WorkSchedule = {
   law_limit_weekly_hours: number;
   law_rest_period_minimum_hours: number;
   law_rest_period_minimum_weekly_hours: number;
+  type?: string;
```

Comment:
Should we use an `enum` here or do we agree on `string`?  
If `string`, I would at least have some known values there for the autocompletion.  

Like `type?: 'known type' | string;`

Reply:
I've implemented the enum on our side; The reason is, I have no firm idea about how it's going to be modified in the future. 
If it's a must, let me know so I can transfer the enum here.

Reply:
I guess it is fine for now, but in the future as the types are stable, we can move it here!  

Thank you for mentioning.